packages/ui/components/form-core/translations/bg.js

```diff
@@ -1,6 +1,6 @@
 export default {
-  validationError: 'Грешка',
-  validationWarning: 'Предупреждение',
-  validationSuccess: 'Успех',
-  validationInfo: 'Информация',
+  validationError: 'Грешка,',
```

Comment:
Maybe it's cleaner to add this inside `.validation-feedback__type`?

Reply:
That was my first thought, @ryubro suggested to put it inside the translations :)
https://github.com/ing-bank/lion/pull/2471#discussion_r1942621870